docs/network/smart-rollup-nodes.md

```diff
@@ -217,3 +209,52 @@ For example, this query gets the health of the node:
    ```
 
 Now that you have a Smart Rollup node configured for Etherlink, you can run an Etherlink EVM node, as described in [Running an Etherlink EVM node](/network/evm-nodes).
+
+## Stopping the Smart Rollup node
+
+When you want to stop running the Smart Rollup node, you must wait for the last commitment that your node made to be cemented.
```

Comment:
If you run in observer (or batcher) it's not necessary to wait. It's implied in your sentence, but maybe it could be made explicit ?
```suggestion
When you want to stop running the Smart Rollup node, you must wait for the last commitment that your node made to be cemented if you had any.
```